.github/workflows/tests.yml

```diff
@@ -0,0 +1,41 @@
+---
+name: Test collection
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  ansible-lint:
+    uses: ansible-network/github_actions/.github/workflows/ansible-lint.yml@main
+  changelog:
+    uses: ansible-network/github_actions/.github/workflows/changelog.yml@main
+  sanity:
+    uses: ansible-network/github_actions/.github/workflows/sanity.yml@main
+  unit-galaxy:
+    uses: ansible-network/github_actions/.github/workflows/unit_galaxy.yml@main
+  unit-source:
+    uses: ansible-network/github_actions/.github/workflows/unit_source.yml@main
```

Comment:
We need a way to disable the tests for old Ansible versions.  We already dropped support for Ansible 2.9.  From experience, trying to support versions of Ansible which are dead upstream prevents us using features of ansible-test, and ansible-core which make our lives simpler.

We also need a way to disable different versions of these tests based on the branch,

Reply:
It's clunky because the default list is so large, but you can specify the `matrix_exclude` input to add `{"ansible-version": "stable-2.9"}` (and also set `matrix_include` to `"{}"` probably once https://github.com/ansible-network/github_actions/pull/47 is available).